src/openrc/rc.c

```diff
@@ -150,6 +151,12 @@ cleanup(void)
 		p1 = p2;
 	}
 
+	for (p1 = LIST_FIRST(&free_these_pids); p1; /* no-op */) {
```

Comment:
Any reason for not using `LIST_FOREACH_SAFE` ? Since we bundle our own `queue.h` there should be no portability issue, right?

Reply:
Pushed in ae12fab, can revert it if there's any objection to it.